pkg/cvo/status.go

```diff
@@ -259,7 +260,11 @@ func (optr *Operator) syncStatus(original, config *configv1.ClusterVersion, stat
 			case len(validationErrs) > 0:
 				message = fmt.Sprintf("Reconciling %s: the cluster version is invalid", version)
 			case status.Fraction > 0:
-				message = fmt.Sprintf("Working towards %s: %.0f%% complete", version, status.Fraction*100)
+				tasks := make([]string, 0, len(status.Current))
+				for _, task := range status.Current {
```

Comment:
This is going to be too large, and I don't want to show resource names here.

Reply:
> This is going to be too large...

Can you think of away to attach as structured data, so the consumer can make the verbosity/rendering calls?